### ISSUE TITLE
Create browser demo for shublang

### ISSUE THREAD

**Issue by akshayphilar** [scrapinghub/shublang#33]

**Comment by peonone** [comment]
Hi @heylouiz 
I tried to integrate [this branch](https://github.com/scrapinghub/shublang/tree/gh-pages-first-version) into one internal product and it works perfectly.

Could you please add the script to generate the javascript files from python source code?
Then we can just re-run this script once there is any update made to the python files.

Thanks in advance.
cc @akshayphilar

**Comment by heylouiz** [comment]
Hi @peonone 
Sorry I missed this notification..
Unfortunately at the moment is not just a matter of running a script to generate the javascript files.
These files are generated by [pyodide](https://github.com/iodide-project/pyodide), this project compile python modules to webassembly and can be used in Javascript.

I forked the project and got shublang to compile, here is the fork I worked on: https://github.com/heylouiz/pyodide

I haven't worked on this for a long time, my last contribution to this project was to add `lxml` (one of Shublang requirements) to the main pyodide project, this was approved last month.
So the next step is to try to integrate shublang too, but I am not sure if they will accept this one day...

So, to summarize:
At the moment it is not possible to simply run a script to generate the files, pyodide project still does not have a script to compile an specific package, you need to clone the whole project and build everything...
There is an [open issue regarding this](https://github.com/iodide-project/pyodide/issues/604), once this kind of tool is available we can do what you asked.

Let me know if you have any questions.
